Assets/Code/Common/Fighter.cs

```diff
@@ -11,4 +11,54 @@ public abstract class Fighter {
     // All fighters have attack and attackTimes (even if they are zero)
     public int AttackValue { get; set; }
     public int AttackTimes { get; set; }
+
+    // Deals with any statuses that exist for any amount of time (e.g. not an instantaneous, 1-time effect)
+    // Key - status type
+    // Value - list of modifiers for a specific StatusType
+    public Dictionary<StaticEffect.StatusType, List<AttachedStatus>> statuses;
```

Comment:
attachedstatus's seem like they should be applied at a higher level than 'figher' as they would be able to attach to any kind of game object, no?

Reply:
same with all the related fields and methods below

Reply:
I agree, but I haven't made a higher level of abstraction than Fighter yet, so it's Fighter for the time being

Reply:
maybe add a TODO on it then?